packages/url_launcher/url_launcher/pubspec.yaml

```diff
@@ -2,7 +2,7 @@ name: url_launcher
 description: Flutter plugin for launching a URL on Android and iOS. Supports
   web, phone, SMS, and email schemes.
 homepage: https://github.com/flutter/plugins/tree/master/packages/url_launcher/url_launcher
-version: 5.7.10
+version: 5.7.10+1
```

Comment:
The next Minor version must be `5.7.11`

Reply:
Update doc can be just bumped by +1  
Example
https://pub.dev/packages/image_picker/changelog#06710
https://pub.dev/packages/url_launcher/changelog#0421 
https://pub.dev/packages/google_maps_flutter/changelog#052113

Also example from my last PR
https://github.com/flutter/plugins/pull/2932/files

Reply:
@TahaTesser 

For package versions > 1.0.0, we shouldn't do +z .

Details in https://dart.dev/tools/pub/versioning#semantic-versions

```
Running version check for changed packages
packages/url_launcher/url_launcher/pubspec.yaml incorrectly updated version.
HEAD: 5.7.10+1, master: 5.7.10.
Allowed versions: {6.0.0: NextVersionType.BREAKING_MAJOR, 6.0.0-nullsafety: NextVersionType.MAJOR_NULLSAFETY_PRE_RELEASE, 5.8.0-nullsafety: NextVersionType.MINOR_NULLSAFETY_PRE_RELEASE, 5.8.0: NextVersionType.MINOR, 5.7.11: NextVersionType.PATCH}
```

Reply:
That's interesting,  made the changes required. 
Thank you